qp_target_gene/split_libraries/util.py

```diff
@@ -41,7 +43,39 @@ def get_artifact_information(qclient, artifact_id):
     # Get the artifact metadata
     prep_info = qclient.get('/qiita_db/prep_template/%s/'
                             % artifact_info['prep_information'][0])
-    qiime_map = prep_info['qiime-map']
+
+    df = pd.read_csv(
+        prep_info['prep-file'], sep='\t', index_col=0, dtype='str')
```

Comment:
Same comments from the other PR regarding data type inference.

Reply:
K